docs/en/Running-HBase-on-Alluxio.md

```diff
@@ -137,3 +137,23 @@ bin/hbase org.apache.hadoop.hbase.mapreduce.RowCounter test
 After this mapreduce job finishes, you can see a result like this:
 
 ![HBaseHadoopOutput]({{site.data.img.screenshot_hbase_hadoop_output}})
+
+## InvalidArgumentException: Offsets do not match
+
+If you see errors in HBase region server log look like:
+
+```
+ERROR alluxio.client.block.stream.NettyPacketWriter: Exception is caught when writing block -1 to channel:
+alluxio.exception.status.InvalidArgumentException: Offsets do not match
+```
+
+Please add the following configuration property to `hbase-site.xml`:
```

Comment:
Could you also add a bit more information as to why this might happen? For example, you could add something like:

This is most likely due to the hbase wal sync using the Alluxio client in a thread unsafe way, but allowing  multiple threads to flush the same Alluxio file stream.

Reply:
I am good with that. We should move this config to the real configure section, and in that section we should briefly explain why this should be set.

Reply:
Will https://github.com/Alluxio/alluxio/pull/7811 fix this issue?

Reply:
Moved to default property section.